Make new icons more consistent with old ones

adamwight opened the pull request:
* Map old to new names

adamwight committed:
Map icon names to newer names

thiemowmde commented:
Note the way this is implemented assumes that the new icon files from version 7.0.0 (on the right side of the aliases table) are available, and the old ones from 0.5.0 are gone, even if this codebase here is technically still the old version. In other words: This patch depends on #1.

As long as this assumption holds this patch is good to go.

thiemowmde commented on index.js:
```diff
@@ -17,6 +17,25 @@ var offsets = {
+var aliases = {
+    'america-football': 'american-football',
+    'camera': 'attraction',
+    'chemist': 'college',
```
I wasn't happy with the ambiguity of the "chemist" icon, [dug deeper and found](https://phabricator.wikimedia.org/T300040#7707095) that it was most certainly meant to represent a pharmacy.

adamwight committed:
Drop xtend dependency

This module was deprecated by es6 syntax.

adamwight committed:
bump minor version for maki 7 upgrade

Maintains backwards compatibility so only bumping the minor.

adamwight committed:
Render at smaller sizes but eliminate margin

The old (maki 0.5.0) icons were rendered with a large margin which is
wasted space in a bitmap.  Instead, this patch renders the full
15x15 (units) new icons in smaller boxes with no margin.  Exact sizes
were chosen such that new icons are roughly congruent to the old ones.

There is a slight savings in the number of bits for each image.

adamwight committed:
Update "chemist" alias

Background by thiemowmde:

"chemist" is heavily ambiguous. Could be a science lab on a
campus. Could be an industrial site. Could be an alternative for a
pharmacy.  A quick image search leaves the impression that no country
uses a test glass to represent a pharmacy. The Maki 2 announcement
mentions Noun Project as inspiration. Searching that for "chemist"
and "pharmacy" doesn't make the situation much clearer.  The patch
that added the icon associates it with many different things.
dewikivoyage translates it as "Drogerie".  A Maki 2 issue from 2016
associates it with "pharmacy".

There are many possible replacements for "minefield", e.g.  "barrier"
or "caution". "danger" felt most appropriate.

"london-underground", "rail-above" and "rail-underground" become
"rail". I couldn't find a better alternative.

adamwight committed:
Rerender symbol pngs

Includes a few more symbols.

Not entirely sure if there were changes, but this set has been
verified to match the maki 0.5 shapes.

adamwight committed:
Verbose filename during render

adamwight committed:
Bump major version to 4 to reflect Maki update

adamwight committed:
Cache the new image sizes

adamwight commented:
This patch updates all the symbols, makes them usable, aligns and sizes to match Maki 0.5, and produces PNG files that can be seamlessly used with Wikimedia's kartotherian installation.

The patch does _not_ fix alphanumeric markers, yet.
